addons/hr_holidays/hr_holidays_view.xml

```diff
@@ -439,6 +441,11 @@
             <field name="view_mode">tree,form,calendar</field>
             <field name="view_id" eval="view_holiday_employee"/>
             <field name="context">{'default_type': 'remove', 'search_default_todo': 1}</field>
+            <field name="help" type="html">
+                <p class="oe_view_nocontent_create">
+                    Click here to create a leave request.
```

Comment:
Missing 'new'
String should be like this "Click here to create a new leave request."